src/pages/Host/HostAmenity/HostAmenity.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import { withRouter } from 'react-router-dom';
+import styled from 'styled-components';
+import axios from 'axios';
+import HostAmenityCom from './HostAmenityCom';
+
+function HostRadio() {
+  const [data, setData] = useState({
+    productList: [],
+  });
```

Comment:
- 그냥 productList 자체를 state 로 사용할 수 있는데, 한번 객체로 감싸는데는 이유가 있을까요? 특별한 이유가 없다면 감싼 객체는 없애주세요! 객체의 구조로 인해 해당 state 를 업데이트 하는 과정이 불필요하게 복잡해집니다!
```suggestion
  const [productList, setProductList] = useState([]);
```